jupyter-book/spatial/imputation.yml

```diff
@@ -0,0 +1,11 @@
+name: imputation
```

Comment:
Is this environment only for the imputation part or also for deconvolution etc? Can we also add environments for these?